### DISCUSSION TITLE
Do something every X seconds

### DISCUSSION THREAD

**Discussion by Android-X13** [arendst/Tasmota#16781] [Q&A] [answered]
This is probably very basic but I can't find the answer in the docs.

Only thing I was able to find is using a rule with a trigger `Time#Minute|x` but I don't see anything equivalent for **seconds**.

**Comment by sfromis** [comment, marked as answer]
You can use a RuleTimer1 (or another number), and have the rules activated when the trigger `Rules#Timer=1` pops start a new RuleTimer1 for the same interval. If your device is ESP32, you also have good opportunities automating via Berry.

**Reply by Android-X13** [reply]
Can you give an example? I tried this but unsuccessfully:

```
Rule
  ON Event#SomeEvent DO Backlog SomeCommand; RuleTimer1 3 ENDON
  ON Rules#Timer=1 DO SomeCommand ENDON
```

`SomeCommand` executes only two times after SomeEvent is fired...

**Reply by sfromis** [reply]
You are doing nothing to invoke the RuleTimer command again, when it has popped once, and therefore no longer in force. You need that command in the 2nd rule.

**Reply by Android-X13** [reply]
Thanks, it works. So it's basically an infinite loop until you stop the rule timer
